src/fides/config/config_proxy.py

```diff
@@ -124,7 +133,7 @@ def __init__(self, db: Session) -> None:
         self.notifications = NotificationSettingsProxy(db)
         self.execution = ExecutionSettingsProxy(db)
         self.storage = StorageSettingsProxy(db)
-        self.security = SecuritySettingsProxy(db)
+        self.security = SecuritySettingsProxy(db, merge_properties={"cors_origins"})
```

Comment:
i think the _cleanest_ implementation for this would be to use a decorator on the property/attribute declaration rather than specifying this as a set of strings in the constructor, but trying to implement that decorator felt like overkill, especially for a fringe feature (merging values) on top of an already fringe feature (config proxy) that already requires a high level of context!

Reply:
It really feels like this little bit of business logic should belong here somehow:
```python
class SecuritySettingsProxy(ConfigProxyBase):
    prefix = "security"

    # only valid URLs should be set as cors_origins
    # for advanced usage of non-URLs, e.g. wildcards (`*`), the related
    # `cors_origin_regex` property should be used.
    # this is explicitly _not_ accessible via API - it must be used with care.
    cors_origins: List[AnyUrl]
```

Can `merge_properties` be declared there? My Python-fu is a bit weak at this exact moment

Reply:
heh i totally agree, as i mentioned in the PR description, i'd like to declare the merge behavior with a decorator here, it's just...i didn't think it was worth the time and effort 😮 

your nudge is enough for me to tinker around a bit more though, i was probably just being a bit lazy...

Reply:
TIL you can't actually put a decorator on the attribute declaration, but i was able to implement a class level decorator (see https://github.com/ethyca/fides/pull/4584/commits/bfceb0364aa44ecda56adf4dd2c61282d2ad3a76) that i think gets us to a pretty good spot - thanks for nudging on this!